playground/nested-deps/vite.config.js

```diff
@@ -1,6 +1,19 @@
+import path from 'node:path'
+import os from 'node:os'
 import { defineConfig } from 'vite'
 
+const isWindows = os.platform() === 'win32'
+const packageFPath = path.resolve(__dirname, 'test-package-f')
+const ensureSlash = (p) => (p.startsWith('/') ? p : `/${p}`)
+
 export default defineConfig({
+  resolve: {
+    alias: {
+      __F_ABSOLUTE_PACKAGE_PATH__: isWindows
+        ? ensureSlash(packageFPath)
+        : packageFPath,
```

Comment:
It doesn't seem right to ensure a leading slash on windows.

Reply:
I pushed it to see how it works before changing and then I forgot 😬
Removed, lets see how CI works now